Source/DataModels/Components/UI/ComponentVideo.h

```diff
@@ -0,0 +1,102 @@
+#pragma once
+#include "Auxiliar/Generics/Drawable.h"
+#include "DataModels/Components/Component.h"
+#include "Resources/ResourceVideo.h"
+
+struct AVFormatContext;
+struct AVCodecContext;
+struct AVPacket;
+struct AVFrame;
+struct SwsContext;
+
+class ComponentVideo : public Component, public Drawable
+{
+public:
+	ComponentVideo(bool active, GameObject* owner);
+	~ComponentVideo() override;
+	void Init();
+	void SetLoop(bool loop);
+	bool GetLoop();
+	void SetCanBeRotate(bool canRotate);
+	bool GetCanBeRotate();
+	void SetRotateVertical(bool rotateVertical);
+	bool GetRotateVertical();
```

Comment:
In general, getters for booleans use verbs like "is" or "can" instead of "get"
```suggestion
	bool IsLooping();
	void SetCanBeRotate(bool canRotate);
	bool CanBeRotate();
	void SetRotateVertical(bool rotateVertical);
	bool CanRotateVertical();
```

Reply:
Done